test/integration/services/data/test-get-team-caseload-overview.js

```diff
@@ -0,0 +1,39 @@
+const expect = require('chai').expect
+
+const workloadCapacityHelper = require('../../../helpers/data/aggregated-data-helper')
+const getTeamCaseloadOverview = require('../../../../app/services/data/get-team-caseload-overview')
+
+var inserts = []
+
+describe('services/data/get-team-caseload-overview', function () {
+  before(function (done) {
```

Comment:
After our discussion yesterday, can we move away from calling `done()` and instead return the promise so that promises don't cause a timeout. This applies to all other tests.